examples/msi-renamed-to-authorization/index.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright 2016-2017, Pulumi Corporation.  All rights reserved.
+
+import * as azure from "@pulumi/azure";
+
+const resourceGroup = new azure.core.ResourceGroup("aks");
+
+const testIdentity = new azure.msi.UserAssignedIdentity("testIdentity", {
```

Comment:
test is small enough to be nbd, but it might stil be good to mention in a comment here that we're going to switch this from azure.msi to azure.authorization in step 2.

nbd though.